docs/talent-agencies.md

```diff
@@ -123,5 +127,6 @@ ID | The ID of the talent agency to retrieve
 | Field | Type | Description |
 |-------|------|-------------|
 | associations | array of strings | Possible values are: `pma`, `sfaal`, `vda` |
+| employees[].about_me | string | This field is dependent on the locale. If no locale is passed, it defaults to "en". |
```

Comment:
```suggestion
| employees[].about_me | string |Returned as plain text (stripped of any rich text formatting). This field is localized. If the `locale` query parameter is not provided, the API attempts to return the English (`en`) version of `about_me`. To request the `about_me` text in a specific language, use the `locale` query parameter (e.g., `?locale=de`). If the `about_me` content for the requested locale is not available, it will return null.|
```